File: Lab2/src/KI/Kulyk/Lab2/Processor.java
package KI.Kulyk.Lab2;

/**
 * Клас для опису процесора.
 */
public class Processor {
    private String model;
    private int cores;

    public Processor(String model, int cores) {
        this.model = model;
        this.cores = cores;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public int getCores() {
        return cores;
    }

    public void setCores(int cores) {
        this.cores = cores;
    }
}

File: Processor.py
class Processor:
    def __init__(self, model, cores):
        self.model = model
        self.cores = cores

    def get_model(self):
        return self.model

    def set_model(self, model):
        self.model = model

    def get_cores(self):
        return self.cores

    def set_cores(self, cores):
        self.cores = cores